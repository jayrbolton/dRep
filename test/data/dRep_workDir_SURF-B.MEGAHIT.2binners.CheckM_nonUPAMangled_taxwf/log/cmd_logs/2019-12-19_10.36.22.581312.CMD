/usr/local/bin/mash sketch /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.038.fasta -s 1000 -o /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/MASH_files/sketches/chunk_0/Bin.038.fasta
