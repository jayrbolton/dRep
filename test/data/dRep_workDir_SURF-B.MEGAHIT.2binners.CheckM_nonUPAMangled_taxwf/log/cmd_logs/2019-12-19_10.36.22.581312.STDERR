Sketching /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_8bins/Bin.038.fasta...
Writing to /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/MASH_files/sketches/chunk_0/Bin.038.fasta.msh...
